packages/angular/package.json

```diff
@@ -87,7 +87,7 @@
     "lodash-es": "4.17.15",
     "ng-packagr": "11.1.4",
     "rxjs": "6.5.3",
-    "sass": "^1.32.11",
+    "sass": "1.36.0",
```

Comment:
Is there something preventing us from upgrading to the latest version? In the react directory I see we are rolling back the sass version.

Reply:
iirc later version failed to build due to dividing with a slash. This version just throws a warning but doesn't fail the build.

Reply:
I think that division should be supported until the next major version
```
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
```

Reply:
We are currently using the version of Sass that you rolled back in the React repo.